### ISSUE TITLE
Cleanup git repo on FORCE to connect with CCBR

### ISSUE THREAD

**Issue by slsevilla** [CCBR/XAVIER#5]
Problem
- Current repo is linked to dnousome account

Solution
- after change of ownership; update FRCE biowulf pull to reflect correct repo path
    -  /mnt/projects/CCBR-Pipelines/pipelines/Exome-seek
- cleanup other remote branches, as needed

**Comment by slsevilla** [comment]
- [ ] change ownership
- [ ] branch cleanup; merge into frce, dev, main
- [ ] repo on frce is set to the `frce` branch
- [ ] ensure correct path location is set (IE CCBR and not previous locations)

**Comment by slsevilla** [comment]
github version set with this location: /mnt/projects/CCBR-Pipelines/pipelines/exome-seek

previous location can be remove @kopardev: 
- /mnt/projects/CCBR-Pipelines/pipelines/Exome-seek

**Comment by escauley** [comment]
Waiting on updates from Samarth and Darryl

**Comment by kopardev** [comment]
No longer needed with XAVIER v3.0
